admin/client/EditorColorScaleSection.tsx

```diff
@@ -304,6 +349,15 @@ class BinLabelView extends React.Component<{
     }
 }
 
+function populateManualBinValuesIfAutomatic(scale: ColorScale) {
```

Comment:
nit: mark the entire function as `@action` and ditch the `runInAction`?

Reply:
I don't think decorators can be used on functions in TS?

Reply:
Seems like they can't:

> A _Decorator_ is a special kind of declaration that can be attached to a [class declaration](https://www.typescriptlang.org/docs/handbook/decorators.html#class-decorators), [method](https://www.typescriptlang.org/docs/handbook/decorators.html#method-decorators), [accessor](https://www.typescriptlang.org/docs/handbook/decorators.html#accessor-decorators), [property](https://www.typescriptlang.org/docs/handbook/decorators.html#property-decorators), or [parameter](https://www.typescriptlang.org/docs/handbook/decorators.html#parameter-decorators).

https://www.typescriptlang.org/docs/handbook/decorators.html#decorators

Reply:
Ahh, right. Sad.